get-parity.sh

```diff
@@ -1,19 +1,61 @@
 #!/bin/bash
 # Copyright 2017 Parity Technologies (UK) Ltd.
+set -x
 RELEASE="beta"
 ARCH=$(uname -m)
 VANITY_SERVICE_URL="https://vanity-service.parity.io/parity-binaries?architecture=$ARCH&format=markdown"
+DISTRO=$(lsb_release -c -s)
+
+check_alt() {
+    if [ "X${DISTRO}" == "X${2}" ]; then
```

Comment:
Why the `X` prefix?

Reply:
@5chdn It's just an old-school best practive to prevent the value of `$DISTRO` to be interpreted as a parameter to `test` (for which `[` is just a syntactic sugar).
For example, if for some ungodly reason `$DISTRO` would be equal to string `"!"`, it might be interpreted as a negation for the upcoming conditional, resulting in an "incorrect syntax" error in runtime.

Reply:
Interesting!